.github/workflows/releaseNigthly.yml

```diff
@@ -236,3 +236,26 @@ jobs:
       run: $GITHUB_WORKSPACE/.github/scripts/upload_failure_logs.sh
       env:
         PLATFORM_TARGET: ${{matrix.target}}
+
+
+  Trigger_Docker:
+    needs: [Linux, Macos]
+    steps:
+    - name: Checkout code
```

Comment:
I see this step is how it is done in other job. From the code I don't understand the reason not to use the _standard_ checkout action. What is it?

Reply:
It was probably a relic of checkout in a docker image.
This is not the case here, so I've moved it to classic github action.